prov/verbs/src/fi_verbs.c

```diff
@@ -153,6 +153,73 @@ vrb_get_rdma_rai(const char *node, const char *service, uint64_t flags,
 	return ret;
 }
 
+static int vrb_get_sin_rai(const char *node, const char *service, uint64_t flags,
+		      const struct fi_info *hints, struct rdma_addrinfo **rai)
+{
+	struct sockaddr_ib *sib;
+	char gid[INET6_ADDRSTRLEN];
+	uint16_t ps;
+	uint16_t pkey;
+	uint8_t scope_id;
+	uint16_t port = 0;
+	int ret;
+
+	memset(gid, 0, sizeof(gid));
+	sib = calloc(1, sizeof(struct sockaddr_ib));
+
+	ret = sscanf(node,
+	             "[%64[^]]]" /* GID */ ":%" SCNx16 /* P_Key */
+	             ":%" SCNx16 /* port space */
+	             ":%" SCNx8 /* Scope ID */,
+	             gid,
+	             &pkey,
+	             &ps,
+	             &scope_id);
```

Comment:
Please don't one parameter per line.  That just forces developers to scroll more to read the function, making it harder to see the entire picture of what's happening.

Reply:
I removed the line breaks.